README.md

```diff
@@ -32,8 +35,17 @@ We will first present the steps and then provide an example.
 
 _NOTE_: that the orthomosaic must be the file name without any extensions; in other words, leave off the `.tif` when specifying it on the Docker command line.
 
-**For example:**
-The files mentioned in this section can be [downloaded](https://drive.google.com/file/d/1U-P4J2OcrNOkaLi6xCUblXOFet7V6raf/view?usp=sharing)
+
+#### For example: <a name="can_shp_example" />
+
+You can download a sample dataset of files (archived) with names corresponding to those listed here from CyVerse using the following command.
+Be sure to replace **<username>** and **<password>** with your CyVerse username and password.
+```bash
+curl -X GET -u '<username>:<password>' https://data.cyverse.org/dav/iplant/projects/aes/cct/diag/sample-data/scif_test_data.tar.gz > scif_test_data.tar.gz
+gunzip scif_test_data.tar.gz
```

Comment:
Suggestion: Replace `gunzip`, `tar`, and `cp` steps with this:

```bash
mkdir inputs
tar xvzf scif_test_data.tar.gz -C inputs
```

- [ ] Create issue to reduce number of steps for data extraction (non-blocking)

Reply:
https://github.com/AgPipeline/issues-and-projects/issues/241

Reply:
Folder is already created. Updated otherwise